Source/APIClient/Request.swift

```diff
@@ -19,6 +19,7 @@ public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
     static var authToken: String? { get }
+    static var authTokenHeader: String? { get }
```

Comment:
I think it's a question of how much we want to abstract authentication while reducing the work required by the user. Right now we're assuming that the token is passed via the Authorization header, which does not cover 100% of cases but handles all the work for APIs that use that structure. We could potentially offload the inclusion of the token in the request entirely to the user, but then our library doesn't really provide any functionality.

The API you mentioned would still technically be supported, since the protocol also exposes `bodyParameters`, it just wouldn't make use of the built-in `authenticated` flag for automatically including the token in the request header. The user presumably has a reference to the token within their app, and they could manually include it in the body of the appropriate requests.